shared/src/main/scala/StringHelpers.scala

```diff
@@ -120,6 +120,16 @@ object StringHelpers:
     val wrapped = (i + s.length) % s.length
     s.substring(0, wrapped) + s.substring(wrapped + 1)
 
+  /** Get the string representation of a value (opposite of eval) */
+  def repr(v: VAny): String =
+    v match
+      case n: VNum => n.toString
+      case s: String =>
+        "\"" + s.replace("\\", "\\\\").replace("\"", "\\\"") + "\""
```

Comment:
```suggestion
      case s: String => quotify(s)
```
Might as well take advantage of what we already have